src/main/java/org/gridsuite/modification/server/modifications/VoltageLevelCreation.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.modifications;
+
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.iidm.network.Network;
+import org.gridsuite.modification.server.dto.VoltageLevelCreationInfos;
+
+/**
+ * @author walid Sahnoun <walid.sahnoun at rte-france.com>
+ */
+public class VoltageLevelCreation extends AbstractModification {
+
+    private final VoltageLevelCreationInfos modificationInfos;
+
+    public VoltageLevelCreation(VoltageLevelCreationInfos modificationInfos) {
+        this.modificationInfos = modificationInfos;
+    }
+
+    @Override
+    public void apply(Network network, Reporter subReporter) {
+        ModificationUtils.getInstance().createVoltageLevelAction(modificationInfos, subReporter, network);
```

Comment:
Rename by `createVoltageLevel`

Reply:
and make it not public

Reply:
i renamed and i removed the public modifier